include/seqan3/alignment/all.hpp

```diff
@@ -200,7 +200,7 @@
  * which can be used to safe some typing.
  *
  * The edit configuration can be further specialised with following configs:
- *  * Allow maximal number of errors, i.e specify the seqan3::align_cfg::max_error configuration
+ *  * Given a minimal score, i.e specify the seqan3::align_cfg::min_score configuration
```

Comment:
I am not happy with the word given, but I am not sure what else to use... Maybe adjust?